### PR TITLE
Add pinch factor correction to controller pointer

### PR DESCRIPTION
**Pull request by rroux-gpsw**
Finally ! Thanks to @bfouet-gpsw

### CODE REVIEW

**Comment by sw-team-release-gpsw** [comment]
[CLA assistant check](https://cla.gopro.com/sign) <br/>All committers have signed the CLA.

File: src/controllers/ControllerPointer.js [hunk 1]
```diff
@@ -262,7 +272,24 @@ FORGE.ControllerPointer.prototype._pinchMoveHandler = function(event)
...
+    if (tmpFov < fovMin)
+    {
+        this._pinchScaleFactorCorrection = this._pinchStartFov * scale / fovMin;
+    }
+    else if (tmpFov > fovMax)
+    {
+        this._pinchScaleFactorCorrection = this._pinchStartFov * scale / fovMax;
+    }
+
+    tmpFov = this._pinchStartFov * scale / this._pinchScaleFactorCorrection;
```
Comment (aberthet-gpsw) [inline src/controllers/ControllerPointer.js]: Wouldn't it be more intuitive to be the inverse ? Like `tmpFov = this._pinchStartFov / scale * this._pinchScaleFactorCorrection;`. It makes more sense in the movement of the user in my mind, but that's debatable.

Comment (aberthet-gpsw) [inline src/controllers/ControllerPointer.js]: My formula doesn't work, but you get the idea

**Comment by bbaudel-gpsw** [comment]
LGTM on Android and iOS

**Pull request merged**